xformers/components/multi_head_dispatch.py

```diff
@@ -127,6 +127,7 @@ def forward(
         key: Optional[torch.Tensor] = None,
         value: Optional[torch.Tensor] = None,
         att_mask: Optional[torch.Tensor] = None,
+        key_padding_mask: Optional[torch.Tensor] = None,
```

Comment:
do we need `key_padding_mask`? Can't this be obtained through the `att_mask` alone?

Reply:
I removed it in an upcoming version, I'm guessing that @sarthmit had it because fairseq, and it complicates the code which is already branchy enough I think